.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: python
+branches:
+    only:
+        - master
+        - /^\d\.\d+$/
+
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=py27
+    - python: 3.4
+      env: TOXENV=py34
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 3.6
+      env: TOXENV=py36
+
+install:
+    - pip install -U pip tox codecov
+
+env:
+  - KAFKA_BROKER=localhost:9092 ZOOKEEPER=localhost:2181
+
+script:
+    - ./kafka-install.sh

```

Comment:
script name is install-kafka.sh, that can be a reason tests are failing on Travis